forks/artifacts/Dockerfile.centos9

```diff
@@ -0,0 +1,24 @@
+FROM registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.18-openshift-4.12 AS oc
+ARG OS_GIT_VERSION="4.12.0"
+
+WORKDIR /go/src/github.com/openshift/oc
+COPY . .
+
+RUN yum install -y --setopt=skip_missing_names_on_install=False rpm-build createrepo bsdtar krb5-devel rsync
+RUN TMPDIR=$( mktemp -d ) RPMBUILD_DIR=_rpmbuild/ && mkdir -p ${RPMBUILD_DIR}/{BUILD,RPMS,SOURCES,SPECS,SRPMS} && \
+    tar -czf "${TMPDIR}/openshift-clients-0.0.1.tar.gz" --exclude='.git' \
+      --transform='s|^.|openshift-clients-0.0.1|' --exclude='openshift-clients-0.0.1.tar.gz' ./ && \
+    cp ./oc.spec "${RPMBUILD_DIR}/SPECS/" && \
+    cp "${TMPDIR}/openshift-clients-0.0.1.tar.gz" "${RPMBUILD_DIR}/SOURCES/" && \
+    rpmbuild -ba --nodeps --define "_topdir $( pwd )/${RPMBUILD_DIR}" \
+      ${RPMBUILD_DIR}/SPECS/oc.spec --define "os_git_vars OS_GIT_VERSION='${OS_GIT_VERSION}' \
+      OS_GIT_COMMIT='${OPENSHIFT_BUILD_COMMIT::8}' OS_GIT_MAJOR='' OS_GIT_MINOR=''  \
+      OS_GIT_TREE_STATE=''"
+
+FROM release:base
+
+COPY --from=oc /go/src/github.com/openshift/oc/_rpmbuild/RPMS/ /tmp/rpms/
```

Comment:
I don't think so (otherwise, similar issues we discussed in the past days would have been assessed)... we could use TARGETARCH or BUILDARCH though, let me check and I'll change it

Reply:
/hold

Reply:
Reintroduced the RUN layer as discussed...